main.py

```diff
@@ -150,6 +171,74 @@ def add_secret(token, target_repository, secret_name, secret_value):
     else:
         print(f"Secret \"{secret_name}\" already exists in {repo_name}")
 
+def add_dependabot_secret(token, target_repository, secret_name, secret_value):
+    repo_name = target_repository.name
+    repo_owner = "philips-internal"
```

Comment:
i don't think this should be hardcoded here

Reply:
fixed.